content/components/main-nav/accessibility/code.md

```diff
@@ -1,4 +1,46 @@
-<nav class="au-main-nav">
+<nav class="au-main-nav" aria-label="main navigation">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div id="main-nav-default" class="au-main-nav__content">
+          <button
+            aria-controls="main-nav-default"
+            class="au-main-nav__toggle au-main-nav__toggle--open js-focus-me"
+            onClick="return AU.mainNav.Toggle( this )">
+            Menu
+          </button>
+          <div class="au-main-nav__menu">
+            <div class="au-main-nav__menu-inner">
+              <div class="au-main-nav__focus-trap-top"></div>
+              <button
+                aria-controls="main-nav-default"
+                class="au-main-nav__toggle au-main-nav__toggle--close js-focus-me"
+                onClick="return AU.mainNav.Toggle( this )">
+                Close
+              </button>
+              <ul class="au-link-list">
+                <li><a class="js-focus-me" href="#">About</a></li>
+                <li><a class="js-focus-me" href="#">Get started</a></li>
+                <li class="active"><a class="js-focus-me" href="#">Components</a></li>
+                <li><a class="js-focus-me" href="#">Templates</a></li>
+                <li><a class="js-focus-me" href="#">Community</a></li>
+                <li><a class="js-focus-me" href="#">Support</a></li>
+              </ul>
+              <div class="au-main-nav__focus-trap-bottom"></div>
+            </div>
+          </div>
+          <div
+            class="au-main-nav__overlay"
+            aria-controls="main-nav-default"
+            onClick="return AU.mainNav.Toggle( this )">
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</nav>
+
+<nav class="au-main-nav au-main-nav--dark" aria-label="main navigation">
```

Comment:
Nice one I definitely forgot to do this. Make sure you change the `id` so there is no duplicated id's. Also make sure the height is big enough. I made a way to set heights on the iframes so it might need to be changed. 👍

Reply:
I changed the id, the page renders correctly without changing any heights.